lib/Puzzle.js

```diff
@@ -1,19 +1,21 @@
 class Puzzle {
   constructor() {
-    this.puzzleBankArray = [];
+    this.puzzleArray = this.generatePuzzles();
   }
 
-  let puzzleArray = Object.keys(data.puzzles);
-  const getPuzzles = puzzleArray.reduce((acc, puzzle) => {
-  let objs = data.puzzles[puzzle].puzzle_bank;
-  let random = Math.floor(Math.random() * objs.length) + 1;
-  acc.unshift(objs[random])
-  return acc;
-  }, []);
 
-getPuzzles;
+  generatePuzzles() {
+    let puzzleArray = Object.keys(data.puzzles);
+    return puzzleArray.reduce((acc, puzzle) => {
+      let objs = data.puzzles[puzzle].puzzle_bank;
+      let random = Math.floor(Math.random() * objs.length) + 1;
+      acc.unshift(objs[random]);
+      return acc;
+      }, []);
+  }
+
 
-};
+  };
 
```

Comment:
Super stoked that we sticked with making into one method to generate our random puzzles!